### PR TITLE
docs(Radio): refactor, use stateless functional expression and fix a typo

### PR DESCRIPTION
**Pull request by dpkwhan**
…stateless functional expression

### CODE REVIEW

**Comment by levithomason** [comment]
Looks good, are there any more changes you expect to make on this PR?

**Comment by levithomason** [comment]
Whoops, actually I see a few things to note.  I'll make a review.

File: docs/app/Examples/addons/Radio/States/index.js [hunk 1]
```diff
@@ -1,41 +1,39 @@
...
-        />
-      </ExampleSection>
-    )
-  }
-}
+const RadioStatesExamples = () => (
+  <ExampleSection title='States'>
+    <ComponentExample
+      title='Checked'
+      description='A radio can come pre-checked'
```
Comment (levithomason) [inline docs/app/Examples/addons/Radio/States/index.js]: We're including the `.` in all descriptions for consistency.  Could you periods to all descriptions?

File: docs/app/Examples/addons/Radio/States/index.js [hunk 2]
```diff
@@ -1,41 +1,39 @@
...
+        Use{' '}
+        <a href='https://facebook.github.io/react/docs/forms.html#default-value' target='_blank'>
+          <code>defaultChecked</code>
+        </a>
+        {' '}as you normally would to set default form values.
+      </Message>
+    </ComponentExample>
+    <ComponentExample
+      title='Disabled'
+      description='Radioes can be disabled'
```
Comment (levithomason) [inline docs/app/Examples/addons/Radio/States/index.js]: Let's fix this typo while we're at it `Radioes` => `Radios` and also add a period.

File: docs/app/Examples/addons/Radio/States/index.js [hunk 3]
```diff
@@ -1,41 +1,39 @@
...
+      examplePath='addons/Radio/States/RadioExampleDisabled'
+    />
+    <ComponentExample
+      title='Read Only'
+      description='Make the radio unable to be edited by the user'
```
Comment (levithomason) [inline docs/app/Examples/addons/Radio/States/index.js]: Let's add a period here as well.

File: docs/app/Examples/addons/Radio/Types/index.js [hunk 4]
```diff
@@ -8,23 +8,23 @@ const RadioTypesExamples = () => (
   <ExampleSection title='Types'>
     <ComponentExample
       title='Radio'
-      description='A radio for checking.'
-      examplePath='addons/Radio/Types/Radio'
+      description='A radio for checking'
```
Comment (levithomason) [inline docs/app/Examples/addons/Radio/Types/index.js]: Let's restore these periods.

File: docs/app/Examples/addons/Radio/Types/index.js [hunk 5]
```diff
@@ -8,23 +8,23 @@ const RadioTypesExamples = () => (
...
+      examplePath='addons/Radio/Types/RadioExampleRadio'
     />
     <ComponentExample
       title='Toggle'
-      description='A radio can toggle.'
-      examplePath='addons/Radio/Types/Toggle'
+      description='A radio can toggle'
```
Comment (levithomason) [inline docs/app/Examples/addons/Radio/Types/index.js]: Let's restore these periods.

File: docs/app/Examples/addons/Radio/Types/index.js [hunk 6]
```diff
@@ -8,23 +8,23 @@ const RadioTypesExamples = () => (
...
-      description='A radio can look like a slider.'
-      examplePath='addons/Radio/Types/Slider'
+      description='A radio can look like a slider'
+      examplePath='addons/Radio/Types/RadioExampleSlider'
     />
     <ComponentExample
       title='Radio Group'
-      description='Radios can be part of a group.'
-      examplePath='addons/Radio/Types/RadioGroup'
+      description='Radios can be part of a group'
```
Comment (levithomason) [inline docs/app/Examples/addons/Radio/Types/index.js]: Let's restore these periods.

File: docs/app/Examples/addons/Radio/Types/index.js [hunk 7]
```diff
@@ -8,23 +8,23 @@ const RadioTypesExamples = () => (
...
-      description='A radio can toggle.'
-      examplePath='addons/Radio/Types/Toggle'
+      description='A radio can toggle'
+      examplePath='addons/Radio/Types/RadioExampleToggle'
     />
     <ComponentExample
       title='Slider'
-      description='A radio can look like a slider.'
-      examplePath='addons/Radio/Types/Slider'
+      description='A radio can look like a slider'
```
Comment (levithomason) [inline docs/app/Examples/addons/Radio/Types/index.js]: Let's restore these periods.

**Review by levithomason — CHANGES_REQUESTED** [review]
One typo fix request and restoring of periods in descriptions.  We recently decided to just keep the period in all example descriptions for consistency.  The issue was that some had one, and others did not.  It was especially confusing when there were multiple sentences.

File: docs/app/Examples/addons/Radio/States/index.js (hunk 1 above)
Comment (dpkwhan) [reply docs/app/Examples/addons/Radio/States/index.js]: Just take a look at some SUI descriptions. All of them are one sentence description.

Comment (levithomason) [reply docs/app/Examples/addons/Radio/States/index.js]: We have several examples with multiple sentences.  You can reference https://github.com/Semantic-Org/Semantic-UI-React/pull/584#discussion_r82046546 for more info.

File: docs/app/Examples/addons/Radio/Types/index.js (hunk 4 above)
Comment (dpkwhan) [reply docs/app/Examples/addons/Radio/Types/index.js]: @levithomason updated with trailing . and fix the typo

**Review by levithomason — APPROVED** [review]

**Comment by levithomason** [comment]
Thanks much!

**Pull request merged**